CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contribution guidelines
+
+We love contributions! We've compiled these docs to help you understand our contribution guidelines.
+
+## Contents of this file
+
+### For contributors
+
+- [Code of conduct](#code-of-conduct)
+- [Conventions to follow](#conventions-to-follow)
+  - [Indentation and whitespace](#indentation-and-whitespace)
+  - [Coding style](#coding-style)
+- [Supported browsers](#supported-browsers)
+- [Supported assistive technology](#supported-assistive-technology)
+- [Commit hygiene](#commit-hygiene)
+
+## Code of Conduct
+
+Please read [the `alphagov` CODE_OF_CONDUCT.md](https://github.com/alphagov/.github/blob/main/CODE_OF_CONDUCT.md) before contributing.
+
+## Conventions to follow
+
+### Indentation and whitespace
+
+2-space, soft-tabs only. No trailing whitespace. The repository contains an .editorconfig file to enforce this if your editor supports this.
+
+### Coding style
+
+We follow the style conventions detailed in [the GDS Way](https://gds-way.cloudapps.digital/manuals/programming-languages.html).
+
+## Supported browsers
+
```

Comment:
There's no harm in having the Supported browsers/assistive technoogy listed here, but it's possibly unlikely to be required on this repo

Reply:
Ah yeah that's true I can probably delete the browser bits